### PR TITLE
fix safari/ie - attempted to assign to readonly property

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@yann300 Also the web3 mode must be updated with this change.

**Comment by yann300** [comment]
in remix you mean? I think that's fine

**Comment by axic** [comment]
No I mean in VM mode the callback still returns the old way so the code breaks?

**Comment by yann300** [comment]
oh yes. I pushed

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
     "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
-    "build": "mkdir -p build; browserify src/index.js -g yo-yoify -o build/app.js; babel --plugins babel-plugin-transform-es2015-template-literals,babel-plugin-transform-es2015-block-scoping build/app.js --out-file build/app.js",
```
Comment (axic) [inline package.json]: Is this a typo or why to remove them?

**Pull request merged**